spec/fixtures/requests/idn.txt

```diff
@@ -11,7 +11,6 @@ Accept-Ranges: bytes
 <html>
 	<head>
 		<meta charset="UTF-8">
-		<meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1" />

```

Comment:
I think it is unnecessary.

Reply:
@ykzts Do you mean this should be replaced to `<meta http-equiv="X-UA-Compatible" content="IE=edge" />`?(Google Chrome Frame is not supported)

Reply:
This file is used for testing only. It simulates the contents of an external web page and has nothing to do with Mastodon.